includes/wp-admin/class-admin.php

```diff
@@ -482,6 +491,10 @@ public static function handle_bulk_request( $sendback, $action, $users ) {
 			$user = new \WP_User( $user_id );
 			if ( 'add_activitypub_cap' === $action ) {
 				$user->add_cap( 'activitypub' );
+
+				\update_user_option( $user_id, 'activitypub_mailer_new_dm', 1 );
```

Comment:
same as above: I would prefer to have a proper default instead of adding the default to the db.

Reply:
Unfortunately, user options don't have the same defaults mechanism that options have, but I could probably build something similar with the filter in get_user_option(). Will give that a try.

Reply:
Updated in [e208e0d](https://github.com/Automattic/wordpress-activitypub/pull/1586/commits/e208e0dcdbedcf9bdd29c89615e975f550352ca9)